batavia/vendor/VENDORING.md

```diff
@@ -0,0 +1,69 @@
+External JavaScript Dependencies
+================================
+
+External JavaScript dependencies live in the `batavia.vendored` namespace. The
+`batavia/vendor/requirements.js` file contains a list of the currently available
+dependencies. Batavia developers consuming already vendored dependencies don't
+need to know anything else or install extra tools.
+
+The code for the vendored dependencies is bundled in the
```

Comment:
Thank you for this detailed response. With this reasoning, I completely agree. 

Linking to this response is something I'll be doing when I find the right place for it in the doc guide